app/models/employee.rb

```diff
@@ -12,14 +12,55 @@ class Employee < ApplicationRecord
   validates :date_of_birth, presence: true, timeliness: { before: 18.years.ago }
   validates :place_of_birth, presence: true, format: { with: /\A[a-zA-Z\s]+\z/ }
   validates :home_address, presence: true, format: { with: /\A\d+\s[A-z0-9]+\s[A-z]+\z/ }
-  validate :employed_on_in_past
```

Comment:
On the model side, we basically save the logic for CRUD operations. For some additional data, and methods, we can create a decorator or a service. So clean up the file and leave here everything related to CRUD.